yarn.lock

```diff
@@ -24559,7 +25203,7 @@ uuid@3.3.2:
   resolved "https://registry.yarnpkg.com/uuid/-/uuid-3.3.2.tgz#1b4af4955eb3077c501c23872fc6513811587131"
   integrity sha512-yXJmeNaw3DnnKAOKJE51sL/ZaYfWJRl1pK9dr19YFCu0ObS231AB1/LbqTKRAQ5kw8A90rA6fr4riOUpTZvQZA==
 
-uuid@^3.0.1, uuid@^3.1.0, uuid@^3.3.2, uuid@^3.4.0:
+uuid@^3.0.0, uuid@^3.0.1, uuid@^3.1.0, uuid@^3.3.2, uuid@^3.4.0:
   version "3.4.0"
   resolved "https://registry.npmjs.org/uuid/-/uuid-3.4.0.tgz#b23e4358afa8a202fe7a100af1f5f883f02007ee"
   integrity sha512-HjSDRw6gZE5JMggctHBcjVak08+KEVhSIiDzFnT9S9aegmp85S/bReBVTb4QTFaRNptJ9kuYaNhnbNEOkbKb/A==
```

Comment:
About `yarn.lock` changes, I only notice additions. An entry for `aws-sdk` still exists. Perhaps if you run `yarn remove ...` to remove the old deps, it should update the `yarn.lock` as well.

Reply:
After checking, it's normal that the dependency is still present since `catalog-backend` uses it in `plugins/catalog-backend/src/ingestion/processors/AwsOrganizationCloudAccountProcessor.ts`:

```ts
import AWS, { Organizations } from 'aws-sdk';
import { Account, ListAccountsResponse } from 'aws-sdk/clients/organizations';
```

Maybe we should update the sdk version (`@aws-sdk/client-organizations`) here too to stay consistent ?
Do you mind if I update it ?

Reply:
For the version upgrade in catalog-backend, I think @backstage/maintainers can help us make the call!

Reply:
Yeah @ayshiff if you want to contribute that in a separate PR we'd definitely appreciate it!